File: src/cxx/VulkanContext/Exceptions/BuilderNotPopulatedException.cpp
//
// Created by kgaft on 7/26/23.
//

#include "BuilderNotPopulatedException.h"

const char *BuilderNotPopulatedException::what() {
    return message;
}

BuilderNotPopulatedException::BuilderNotPopulatedException(const char *message) : message(message) {}

File: src/main/java/com/kgaft/VulkanContext/Exceptions/BuilderNotPopulatedException.java
package com.kgaft.VulkanContext.Exceptions;

public class BuilderNotPopulatedException extends Throwable {
    public BuilderNotPopulatedException(String message) {
        this.message = message;
    }

    private String message;

    @Override
    public String getMessage() {
        return message;
    }
}
